### ISSUE TITLE
what's your learning rate decaying strategy

### ISSUE THREAD

**Issue by LiZhenLiangLee** [ialhashim/DenseDepth#112]
Hi, I know your  initial learning rate is 0.0001. I wonder how do you decay the lr ?
And I started training your keras model  ( `python train.py` ),  but I am not familiar with tensorfow and keras. I wonder if you do lr decaying in your code, or you just use initial lr in all the training process?

**Comment by ialhashim** [comment]
You can find the learning rate decay [here](https://github.com/ialhashim/DenseDepth/blob/master/callbacks.py#L78).

**Comment by LiZhenLiangLee** [comment]
> You can find the learning rate decay [here](https://github.com/ialhashim/DenseDepth/blob/master/callbacks.py#L78). There are other types of scheduling you can look at for Keras.

Thanks for reply
